charts/vsphere-cpi/v1.0.0/templates/vsphere-creds-secret.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: v1
```

Comment:
why are we removing this option? also (if we bring this back), instead of stringData we should probably use b64enc, as seen in https://github.com/prometheus-community/helm-charts/blob/4ac76c1bd53e92b61fe0e8a99c184b35e471cede/charts/prometheus-pingdom-exporter/templates/secret.yaml#L12-L16.